src/lib.rs

```diff
@@ -1,10 +1,14 @@
-use anyhow::Result;
+//! Our module that implements the dials research app.`
```

Comment:
nit: OUR MODULE lol, also whats with the rouge "`"
Welcome to the indian coding toturial, copy down the following code with me:
```
int myVariable = 10;
printf("%i", myVariable);
```

Reply:
I turned on #[deny(missing_docs)] for a while, lol